### ISSUE TITLE
Metadata text fields gets reset after editing permissions

### ISSUE THREAD

**Issue by helbashandy** [NCEAS/metacatui#1890]
**Describe the bug**
When editing the metadata fields then editing the metadata file's sharing permissions, the edits on the metadata page gets removed (reset) 

**To Reproduce**
1. Go to https://dev.nceas.ucsb.edu/data
2. Create a data package and submit, then go to edit view.
3. Write anything in the package title field. 
4. Change the metadata file permissions through the share button.
5. Note that your title changes are now gone. 

**Expected behavior**
Metadata changes should not be lost when editing permissions.

**Comment by laurenwalker** [comment]
Yes there is a linked commit from today that closed this issue. It is resolved with that commit.
